Mathlib/Data/Nat/Factorial/BigOperators.lean

```diff
@@ -37,4 +38,43 @@ theorem descFactorial_eq_prod_range (n : ℕ) : ∀ k, n.descFactorial k = ∏ i
   | 0 => rfl
   | k + 1 => by rw [descFactorial, prod_range_succ, mul_comm, descFactorial_eq_prod_range n k]
 
+lemma exists_descFactorial_eq_polynomial (k : ℕ) : ∃ (a : Fin k → ℕ), ∀ n,
+    (n + k).descFactorial k = n ^ k + ∑ i, a i * n ^ (i : ℕ) := by
```

Comment:
Does it make sense to define this as a `Polynomial Nat`?

Reply:
Indeed, it's ascPochhammer up to a shift, so I have deleted this and used directly ascPochhammer in the main proof. Thanks!